token.go

```diff
@@ -49,6 +49,20 @@ const (
 	doneSrvError = 0x100
 )
 
+// CurCmd values in done (undocumented)
+const (
+	cmdSelect = 0xc1
+	// cmdInsert     = 0xc3
```

Comment:
Are all these meant to be commented out?

Reply:
yes, otherwise lint complains about unused private symbols. I have them there for convenience should we ever need to inspect the tokens.